source/backend/entities/Partials/ProjectStatusType.cs

```diff
@@ -27,6 +22,15 @@ public PimsProjectStatusType(string id)
         {
             Id = id;
         }
-        #endregion
+
+        public static class PimsProjectStatusTypes
```

Comment:
I don't see this being used anywhere, why is this here?

Reply:
removed